_posts/2020-10-23-autonomy.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: Enabling autonomous teams while maintaining direction
+tags: [tech-debt, scale]
+comments: true
+---
+
+
+<!-- TODO: write teaser -->
+<!-- Autonomy and collaboration are two buzzwords in engineering management, and rightly so; they are both
```

Comment:
I liked this post, but I get what you mean when you say you think it might be stating what is obvious.

Firstly, I don't think it's that obvious, and I'm sure others would find it useful anyway.

But I reckon you're missing the point of who your audience is? I suspect this was prompted by a conversation with a group of TLs who are struggling to find this balance, and understand what autonomy means to them, in their teams.

- Why do you think they're struggling?
- Is there something you can share with them that might make them feel better?

If you think their struggle is because they have the wrong definition of autonomy (I can do whatever I want), then maybe you have some ideas about why they shouldn't be worried, coming from how you define it? Is it because they hear autonomy is important, but don't have a good sense of what decisions are crucial for the team to be making, vs those that are well set from above?

If you can pinpoint your audience, and address a struggle that most team leads encounter as they grow, then I think this could be really useful.

Reply:
thanks for this feedback! I've finally got around to revisiting this, I've added a pgraph in the middle which I hope helps address this a bit more clearly.